### PR TITLE
[OCP/Heterogeneous] Enabling QE tests on AWS

### PR DESCRIPTION
**Pull request by aleskandro**

### CODE REVIEW

**Comment by aleskandro** [comment]
The installations passed for both the profiles

**Comment by lwan-wanglin** [comment]
thanks @aleskandro 
/lgtm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @heterogeneous and @network-openshiftsdn and not
+        @fips'
+      TAG_VERSION: '@4.12'
+    test:
+    - ref: ipi-install-heterogeneous
```
Comment (LiangquanLi930) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:64]: I think `ipi-install-heterogeneous` should belong to workflow, not test

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+tests:
+- as: e2e-aws-ipi-p2
+  cron: 0 6 */3 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @heterogeneous and @network-openshiftsdn and not
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:60]: `network-openshiftsdn` should be `network-ovnkubernetes`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: e2e-aws-ipi-p2
+  cron: 0 6 */3 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:53]: Based on the rules we currently have, p2 means every week (*/7).

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml (hunk 1 above)
Comment (aleskandro) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml:64]: Hi @LiangquanLi930 , I prefer to do that as a follow up. The step `ipi-install-heterogeneous` is still in an early phase. This PR is meant to provide a baseline of heterogeneous (Technological Preview for 4.12) testing on prow

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml (hunk 3 above)
Comment (aleskandro) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml]: Thanks @liangxia done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml (hunk 2 above)
Comment (aleskandro) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml]: Thanks @liangxia done

**Pull request merged**